integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/iterate/IterateJsonPathTest.java

```diff
@@ -0,0 +1,194 @@
+package org.wso2.carbon.esb.mediator.test.iterate;
+
+import org.apache.commons.io.FileUtils;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.exceptions.AutomationFrameworkException;
+import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.ProtocolException;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * This class tests Iterate Mediator Jsonpath support with Json Payloads
+ */
+public class IterateJsonPathTest extends ESBIntegrationTest {
+
+    private String input;
+
+    @BeforeClass(alwaysRun = true)
+    public void uploadSynapseConfig() throws Exception {
+        super.init();
+        input = FileUtils.readFileToString(new File(getESBResourceLocation() + "/json/inputESBIterateJson.json"));
```

Comment:
Make sure hardcoding "/" does not crash in Windows environment

Reply:
Will this not cause an issue in windows?

Reply:
No. Tested and confirmed.

Reply:
Fixed in https://github.com/wso2/product-ei/pull/4782